### PR TITLE
Installation using helm in website

### LINKED ISSUE #753: [DOC] add helm chart support to installation
We can install kueue via helm chart https://github.com/kubernetes-sigs/kueue/pull/664 now.
/kind documentation
cc @Gekko0114

### PR DESCRIPTION
**Pull request by BinL233**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
Added kueue installation guide using helm in website. The content was used from [README](https://github.com/kubernetes-sigs/kueue/blob/main/charts/kueue/README.md)
#### Which issue(s) this PR fixes:

Fixes #753

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by BinL233** [comment]
cc @kerthcet

File: site/content/en/docs/installation/_index.md [hunk 1]
```diff
@@ -205,3 +205,47 @@ To uninstall Kueue, run the following command:
 ```sh
 make undeploy
 ```
+## Install kueue using Helm
```
[the remark this answers is not in the document]
Comment (BinL233) [reply site/content/en/docs/installation/_index.md]: @alculquicondor Thanks for your comment. I just created a new file which linked to index page and delete /charts/kueue/README.md.

File: site/content/en/docs/installation/_index.md [hunk 2]
```diff
@@ -205,3 +205,7 @@ To uninstall Kueue, run the following command:
 ```sh
 make undeploy
 ```
+## Install Kueue with Helm
+
+To install and configure Kueue with Helm, follow [instructions](/site/content/en/docs/installation/Helm.md).
```
Comment (kerthcet) [inline site/content/en/docs/installation/_index.md:210]:
```suggestion
To install and configure Kueue with [Helm](https://helm.sh/), following [instructions](/docs/installation/Helm.md).
```

File: site/content/en/docs/installation/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kerthcet) [reply site/content/en/docs/installation/_index.md]: hm.. I think we should keep the README.md for helm chart, it's part of the chart, once we upload it to the market, people can refer to this document.

[the remark this answers is not in the document]
Comment (alculquicondor) [reply site/content/en/docs/installation/_index.md]: oh, I see.
What if we keep it but it just has a link to the website? That way, we don't have to maintain 2 files.

[the remark this answers is not in the document]
Comment (kerthcet) [reply site/content/en/docs/installation/_index.md]: make sense to me

File: charts/kueue/README.md [hunk 3]
```diff
@@ -3,12 +3,14 @@
 ## Table of Contents
 
 <!-- toc -->
-- [Installation](#installation)
-  - [Prerequisites](#prerequisites)
-  - [Installing the chart](#installing-the-chart)
-    - [Install chart using Helm v3.0+](#install-chart-using-helm-v30)
-    - [Verify that controller pods are running properly.](#verify-that-controller-pods-are-running-properly)
-  - [Configuration](#configuration)
+- [Kueue's helm chart](#kueues-helm-chart)
```
Comment (kerthcet) [inline charts/kueue/README.md:6]: Why need t his line? Also the below one.

File: site/content/en/docs/installation/_index.md [hunk 4]
```diff
@@ -205,3 +205,6 @@ To uninstall Kueue, run the following command:
 ```sh
 make undeploy
 ```
+## Install Kueue with [Helm](https://helm.sh/)
+
+To install and configure Kueue with Helm, follow the [instructions](/charts/kueue/README.md).
```
Comment (alculquicondor) [inline site/content/en/docs/installation/_index.md:210]: This link won't work.

I think we should do it the other way:
change the README.md file to refer to a page in the website.

Comment (kerthcet) [reply site/content/en/docs/installation/_index.md:210]: We can refer to https://github.com/kubernetes-sigs/kueue/blob/main/charts/kueue/README.md directly.

Comment (kerthcet) [reply site/content/en/docs/installation/_index.md:210]: Can you add an indexer in the beginning?

File: site/content/en/docs/installation/_index.md [hunk 5]
```diff
@@ -205,3 +205,6 @@ To uninstall Kueue, run the following command:
 ```sh
 make undeploy
 ```
+## Install Kueue with [Helm](https://helm.sh/)
```
Comment (kerthcet) [inline site/content/en/docs/installation/_index.md:208]:
```suggestion
## Install via [Helm](https://helm.sh/)
```

**Review by kerthcet — COMMENTED** [review]
Please move the section of `Change the feature gates configuration` to the end of the content.

**Comment by BinL233** [comment]
The new pull request is [here](https://github.com/kubernetes-sigs/kueue/pull/1006)

**Pull request closed without merging**